modules/monitoring/controllers/pnp.php

```diff
@@ -49,9 +49,32 @@ public function pnp_default()
 		if ($pnp_path != '') {
 			$source = intval($this->input->post('source', false));
 			$view = intval($this->input->post('view', false));
+			$settings = json_encode(array("source"=>$source, "view"=>$view));
 
-			Ninja_setting_Model::save_page_setting('source', $pnp_path.'/image?'.$param, $source);
-			Ninja_setting_Model::save_page_setting('view', $pnp_path.'/image?'.$param, $view);
+			Ninja_setting_Model::save_page_setting('default_graph', $pnp_path.'/image?'.$param, $settings);
```

Comment:
Changing the data model for this could potentially break something elsewhere. Are we certain this is not used anywhere in the product today?

Reply:
After doing some digging I cannot find any place where we actually use this setting. I dont think we're gonna break anything.